README.md

```diff
@@ -1,4 +1,27 @@
 # What is ActiveRecord ?
 
-1. What is ActiveRecord? 
-2. ActiveRecord vs. SQL
\ No newline at end of file
+## Objectives
+
+1. Define ActiveRecord and understand why it is easier to use.
+2. Compare ActiveRecord vs. SQL queries.
+
+## ActiveRecord
+
+ActiveRecord is what is referred to as an Object-Relational Mapper (ORM). It's responsible for mapping exactly one row in a database to one object in Ruby. 
```

Comment:
also add something like: "Essentially, ActiveRecord is a wrapper written in Ruby that allows us to interact with a database without having to go through the pain of writing long SQL queries. While SQL queries are still run in sinatra applications, ActiveRecord abstracts the process, making it much easier for us developers to write clean, understandable code."